deployer/deploy_functions.go

```diff
@@ -0,0 +1,144 @@
+package main
```

Comment:
Since this isn't deploying any functtions, wdyt about `/support/codegen`?

Reply:
Moved to support/codegen